db/warehouse/migrate/20230927205059_add_service_indexes.rb

```diff
@@ -0,0 +1,11 @@
+class AddServiceIndexes < ActiveRecord::Migration[6.1]
+  def change
+    safety_assured do
+      add_index :Services, :EnrollmentID
```

Comment:
This is potentially a fairly large hit on some of our production DBs.  Any chance we can add it concurrently?

Reply:
Actually, this shouldn't be necessary, there should already be an index with `"EnrollmentID","PersonalID",data_source_id` which should cover `EnrollmentID` alone since it's first.

Reply:
> This is potentially a fairly large hit on some of our production DBs. Any chance we can add it concurrently?

For sure, I'll make that change before I hand it off

>Actually, this shouldn't be necessary, there should already be an index with "EnrollmentID","PersonalID",data_source_id which should cover EnrollmentID alone since it's first.

Great, I missed that. Thanks!

Reply:
I left this migration as-is since we are only adding indexes to CustomServices which I assume is a relatively small table (please correct me if I'm wrong!). Its less hassle to keep the ddl transaction and not have to deal with partially applied migrations. Happy to move to concurrent index creation if this table is large though

Reply:
looks like that table only has some 10k rows right now so, probably fine.